crypto/fipsmodule/service_indicator/service_indicator_test.cc

```diff
@@ -1967,6 +1967,451 @@ TEST_P(KDF_ServiceIndicatorTest, TLSKDF) {
   EXPECT_EQ(approved, test.expect_approved);
 }
 
+// PBKDF2 test data from RFC 6070.
```

Comment:
maybe put a comment here or in the tests that Salt1 and Password1 are too short and will cause the service indicator to be NOT_APPROVED.
Also a comment about Salt3 and Password3 would be good as I don't know what they add. Aren't they short too?

Reply:
They are; the only difference is that they include `NUL` characters, presumably to ensure the password/salt are being handled as byte buffers instead of C strings. I'll remove set 3 as it doesn't really add anything to AWS-LC.